src/controllers/developers.js

```diff
@@ -62,3 +58,23 @@ export const postDeveloper = async (req, res, next) => {
     return next(err);
   }
 };
+
+export const getDeveloper = async (req, res, next) => {
+  const connection = createDbConnection();
+  const Developer = connection.model('Developer', developerSchema);
+  const devId = req.params.id;
+  try {
+    const developer = await Developer.find({ _id: devId });
```

Comment:
we have a custom Mongoose plugin responsible for applying the `id` field to the found `developer` object. So we would want to do the following to remove the default `_id` key with `id`
```suggestion
    let developer = await Developer.find({ _id: devId });
    
    if (developer) {
      throw new Error('Developer doesn\'t exist');
    }

    developer = developer.toJSON();
```